src/lib/uri/uri.h

```diff
@@ -13,6 +13,15 @@
 extern "C" {
 #endif /* defined(__cplusplus) */
 
+struct uri_query_param {
```

Comment:
I think this structure must be hidden in C file.

Reply:
fixed